frontend/src/routes/+page.svelte

```diff
@@ -1,3 +1,34 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import { get } from '$utils/fetch';
+  import { _ } from 'svelte-i18n';
+  import { goto } from '$app/navigation';
+
+  interface Course {
+    name: string;
+    user: {
+      first_name: string;
+    };
+  }
+  let searchQuery = '';
+  let courses: Course[] = [];
+
+  async function getCourses() {
+    const response = await get('/v1/courses');
+    const data = await response.json<Course[]>();
+    courses = data as Course[];
```

Comment:
You don't need to provide `as Course[];` as you're using Generic. Did you actually read article I've posted or just made this change blindly?